mantidproject/mantid
# Update Qt library names in the OS X packaging script.

Description of work.

This updates the library name to patch what is reported by `otool -L`. OS X 10.9 and 10.10 seem OK with the wrong path, but on 10.11, `install_name_tool` isn't changing the library name, causing a sanity check to fail.

**To test:**



Check if you can make a package on OS X 10.11 and the current OS X build servers still pass.

This issue has no associated issue.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
